HomeEdu/Data/Model/ViewScore/ScoreSubject.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ScoreSubject.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 8/19/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+final class ScoreSubject: Mappable {
+    var midterm: Double?
+    var total: Double?
+    var final: Double?
```

Comment:
```suggestion
    var `final`: Double?
```
Sai này vẫn build đc à em?